### PR TITLE
Modified changelog functionality

### PR DESCRIPTION
**Pull request by curlyLasagna**
Fixes the issue of viewing change logs

## Change log from details page
https://github.com/user-attachments/assets/025055b0-e469-4704-88e0-6c0c328c0c1a

## Change log from dashboard page
https://github.com/user-attachments/assets/ab85ab1f-6d4c-4883-9e54-b09bcd802710

### CODE REVIEW

File: src/app/shared/modules/changelogs/changelog-item/changelog-item.component.html [hunk 1]
```diff
@@ -1,19 +1,14 @@
...
-        ></clark-identification-pill>
-      </div>
-      <div class="changelog-item__date">
-        on {{ changelog.date | date:'shortDate' }}
-      </div>
+      {{ author.name | titlecase }}
+      <!-- the date property is not present with the data in the database -->
+      <!-- <div class="changelog-item__date"> -->
+        <!-- on {{ changelog.date | date:'shortDate' }} -->
+      <!-- </div> -->
```
Comment (Cwagne17) [inline src/app/shared/modules/changelogs/changelog-item/changelog-item.component.html:11]: Nit: Can we remove this if it is not being used

**Review by Cwagne17 — APPROVED** [review]

File: src/app/shared/modules/changelogs/changelog-item/changelog-item.component.ts [hunk 2]
```diff
@@ -1,17 +1,36 @@
...
+      this.user = await this.userService.getUser(this.changelog.authorId, '');
+      this.author = {
+        name: this.user.name,
+        profileImage: await this.userService.getGravatarImage(this.user.email, 200),
+      };
+    } else {
+      this.author = {
+        name: 'CLARK Editor',
+        profileImage: 'assets/images/logo.png'
+      };
```
Comment (Cwagne17) [inline src/app/shared/modules/changelogs/changelog-item/changelog-item.component.ts:31]: I noticed in both videos we have CLARK Editor as the user for the changelog, did you test to make sure that a contributor that is the changelog author appears also?

**Review by Cwagne17 — APPROVED** [review]

File: src/app/shared/modules/changelogs/changelog-item/changelog-item.component.ts (hunk 2 above)
Comment (curlyLasagna) [reply src/app/shared/modules/changelogs/changelog-item/changelog-item.component.ts:31]: image

File: src/app/shared/modules/changelogs/changelog-item/changelog-item.component.html (hunk 1 above)
Comment (pzalep1) [reply src/app/shared/modules/changelogs/changelog-item/changelog-item.component.html:11]: We want to keep this for now, but we should add a comment that this was commented out on this date and we will fix it when we get to changelog updates with tagging.

**Review by pzalep1 — APPROVED** [review]

**Review by johngonz0 — APPROVED** [review]

**Review by Cwagne17 — APPROVED** [review]

**Pull request merged**